03-functions-and-loops/invest.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def invest(amount, rate, num_of_years):
+    initial_amount = amount
+    print("\nPrincipal amount: ", amount)
+
+    print("annual return rate: ", str(rate))
+
+    for year in range(1, num_of_years + 1):
+        compound_rate = amount * rate
+        amount = amount + compound_rate
+        print("year ", num_of_years, ":", amount)
+
+    roi  = amount - initial_amount
```

Comment:
Please refrain from using abbreviations as it might cause ambiguity for someone who's not familiar with the code. And I think the correct term would be `margin`.